web/src/pages/index.tsx

```diff
@@ -86,12 +83,6 @@ const Home = (props: Props): ReactElement => {
       } else if (userData === undefined && error !== undefined) {
```

Comment:
[Dust] Should we remove the else if conditional here and just keep what is inside of the code block?